site/pages/component/gc-table-en.hbs

```diff
@@ -0,0 +1,180 @@
+---
+{
+	"altLangPrefix": "gc-table",
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" },
+		{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
+	],
+	"dateModified": "2020-10-16",
+	"description": "Working examples for tables - provisional feature",
+	"language": "en",
+	"parentdir": "component",
+	"tag": "table",
+	"title": "Tables - Provisional feature"
+}
+---
+{{>alert-provisional-demos}}
+<span class="wb-prettify all-pre hide"></span>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Provisional</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca design pattern</dd>
+	<dt>Last review</dt>
+	<dd>2021-02-01</dd>
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+	<dt>Guidance</dt>
+	<dd><a href="#">Tables - Canada.ca design pattern [coming soon]</a></dd>
+	<dt>Technical name</dt>
+	<dd>
+		<ul>
+			<li><code>gc-table</code></li>
+		</ul>
+	</dd>
+</dl>
+
+<h2>Working example</h2>
+
+<p>Responsive cards</p>
+<table class="provisional gc-table table" id="myTable">
+		<caption class="wb-inv">Population growth in Canadian cities</caption>
+		<thead>
+		<tr>
+			<th scope="col">City</th>
+			<th scope="col">Population in 2007</th>
+			<th scope="col">Population in 2017</th>
+			<th scope="col">Percentage change</th>
+		</tr>
+		</thead>
+		<tbody>
+		<tr>
+			<td data-label="City">Toronto</td>
+			<td data-label="Population in 2007">5,418,207</td>
+			<td data-label="Population in 2017">6,346,088</td>
+			<td data-label="Percentage change">17.1%</td>
+		</tr>
+		<tr>
+			<td data-label="City">Montréal</td>
+			<td data-label="Population in 2007">3,714,846</td>
+			<td data-label="Population in 2017">4,138,254</td>
+			<td data-label="Percentage change">11.4%</td>
+		</tr>
+		<tr>
+			<td data-label="City">Vancouver</td>
+			<td data-label="Population in 2007">2,218,134</td>
+			<td data-label="Population in 2017">2,571,262</td>
+			<td data-label="Percentage change">15.9%</td>
+		</tr>
+		<tr>
+			<td data-label="City">Ottawa–Gatineau</td>
+			<td data-label="Population in 2007">1,188,073</td>
+			<td data-label="Population in 2017">1,377,016 </td>
+			<td data-label="Percentage change">15.9%</td>
+		</tr>
+		</tbody>
+	</table>
+
+<p>Responsive cards with text left on xxs</p>
+<table class="provisional gc-table table" id="myTable">
+		<caption class="wb-inv">Telephone numbers for personal taxes, benefits and trusts</caption>
+		<thead>
+		<tr>
+			<th scope="col">Reason for your call</th>
+			<th scope="col">Number</th>
+			<th scope="col">Hours</th>
+		</tr>
+		</thead>
+		<tbody>
+		<tr>
+			<td data-label="Reason for your call">
+				<span class="left-xxs">
+					<ul>
+						<li>Item one</li>
+						<li>Item two</li>
+					</ul>
```

Comment:
Try to move the ".left-xxs" class to be set on the `<ul>` element and remove the `<span>`